### ISSUE TITLE
Map images in gallery are missing URLs

### ISSUE THREAD

**Issue by jywarren** [publiclab/mapknitter#868]
It looks like something went wrong with the images on the gallery page: https://mapknitter.org/gallery

```html
<img class="card-img-top map-img" src="" alt="law street beach">
```

![image](https://user-images.githubusercontent.com/24359/61542472-a0c4e380-aa0f-11e9-8ca8-5bdb853fa51f.png)

@cesswairimu do you have a moment to look at that? Otherwise I can dig into it. Thanks! Maybe we should write an integration test for it...

**Comment by cesswairimu** [comment]
Hey @jywarren yeah I noticed that...I fixed it in https://github.com/publiclab/mapknitter/pull/777/commits/f56041f489c2cd2843bace22be40e77348d48643 but I can create a PR with that commit if you feel the view only page is not ready yet. Thanks

**Comment by jywarren** [comment]
Oh gotcha - thanks, Cess! I'll catch up on there and we'll see, but I think
it's not hyper urgent! Thanks.

**Comment by cesswairimu** [comment]
Cool, making a pr

**Comment by jywarren** [comment]
oh sorry, i meant we can wait for your view map PR, i think. Sorry, have
been behind due to this conference!

**Comment by cesswairimu** [comment]
No worries @jywarren I understand

**Comment by cesswairimu** [comment]
Aaah! I misread that sorry

**Comment by cesswairimu** [comment]
We fixed in #904. closing this
